.travis.yml

```diff
@@ -2,10 +2,11 @@ language: python
 
 python:
 - '2.7'
+- '3.5'
 
 env:
   - EXTRAS='[test]'
-  - EXTRAS='[test,s3,test-s3]'
+  - EXTRAS='[test,s3,test_s3]'
```

Comment:
Hey @rzimmerman can you advise on why this change is necessary? And why a corresponding change is apparently not needed in setup.py?